CommonSettings.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <PropertyGroup>
+	<RoundsFolder>C:\Program Files (x86)\Steam\steamapps\common\ROUNDS</RoundsFolder>
+	<ManagedFolder>$(RoundsFolder)\ROUNDS_Data\Managed</ManagedFolder>
+	<BepInFolder>../Assemblies</BepInFolder>
```

Comment:
Is there a reason for this to be a variable?
If for whatever reason it is, maybe rename it to "BepInExCoreFolder" or something of the sort.
But realistically i dont see the use in this being changed ever.